### PR TITLE
Rename collect outputs to claim outputs

### LINKED ISSUE #1179: [Task]: rename collect outputs to claim outputs
### Task description

Rename collect to claim

### Requirements

N/A

### Acceptance criteria

N/A

### Creation checklist

- [ ] I have assigned this task to the correct people
- [X] I have added the most appropriate labels
- [X] I have linked the correct milestone and/or project

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Rename collect outputs to claim outputs, because that's clearer

## Links to any relevant issues

Fixes #1179 

## Type of change

- Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Change checklist

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: bindings/nodejs/examples/21-collect-outputs.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ async function handleNewOutputOfBob(err, data) {
     if (event.accountIndex === bob.meta.index) {
         const outputId = event.event.NewOutput.output.outputId
         await bob.sync()
-        const resp = await bob.collectOutputs([outputId])
+        const resp = await bob.claimOutputs([outputId])
```
Comment (Tuditi) [inline bindings/nodejs/examples/21-collect-outputs.js:51]: Could you also rename the file name and the log?

**Review by Tuditi — CHANGES_REQUESTED** [review]
One minor request, but I like the naming much more this way! :relaxed:

File: bindings/nodejs/examples/21-collect-outputs.js (hunk 1 above)
Comment (Thoralf-M) [reply bindings/nodejs/examples/21-collect-outputs.js:51]: Sure!

**Review by thibault-martinez — APPROVED** [review]

**Review by samuel-rufi — APPROVED** [review]

**Review by Tuditi — APPROVED** [review]
LGTM! :rocket:

**Pull request merged**